automation/src/test/scala/org/broadinstitute/dsde/firecloud/config/AuthTokens.scala

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.dsde.firecloud.config
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+
+import scala.collection.JavaConversions._
```

Comment:
I think the best practice is to import `scala.collection.JavaConverters._` instead of `JavaConversions`. The difference is that `JavaConverters` enriches Java/Scala collection classes with `.asJava` and `.asScala` methods, while `JavaConversions` tries to do it for you behind the scenes. It tends to be better to be explicit about when you want a conversion to avoid surprises/unexpected errors.